velox/exec/tests/utils/TpchQueryBuilder.cpp

```diff
@@ -55,6 +55,16 @@ std::string formatDateFilter(
   VELOX_FAIL(
       "Date range check expression must have either a lower or an upper bound");
 }
+
+std::vector<std::string> mergeColumnNames(
+    std::vector<std::string> firstColumnVector,
```

Comment:
const &

Reply:
Thanks for the suggestion, passing the address of firstVector here resulted in redundant column names since the same firstVector variable, lineitemCommonColumns, is used at multiple places in the caller, so const & was not used. 
The unit test added to ParquetTpchTest verifies correctness of the TPCH query plan by comparing the results with that produced by DuckDb, should I add other tests for verifying correctness?

Reply:
> The unit test added to ParquetTpchTest verifies correctness of the TPCH query plan by comparing the results with that produced by DuckDb.

That's nice. We are good then.

Reply:
Thanks for the input, will ensure that input parameters are not modified going forward